src/main/java/com/google/api/gax/grpc/ServiceApiSettings.java

```diff
@@ -219,4 +228,12 @@ public void setGeneratorHeader(String name, String version) {
     this.serviceGeneratorName = name;
     this.serviceGeneratorVersion = version;
   }
+
+  /**
+   * Sets the client library name and version for the GRPC custom header.
+   */
+  public void setClientLibHeader(String name, String version) {
+    this.appName = name;
+    this.appVersion = version;
```

Comment:
be consistent: call the function & variables the same - either both 'clientLib', or both 'app'.